Directory.Build.props

```diff
@@ -153,9 +153,16 @@
 
   <!-- Enable the analyzers for this repo -->
   <PropertyGroup>
-    <EnableAnalyzers Condition="'$(EnableAnalyzers)' == '' AND '$(IsSourceProject)' == 'true'">true</EnableAnalyzers>
+    <EnableAnalyzers Condition="'$(EnableAnalyzers)' == '' AND ('$(IsSourceProject)' == 'true' OR '$(IsTestProject)' == 'true')">true</EnableAnalyzers>
```

Comment:
are there any projects that aren't one of these two types?

Reply:
Yeah, there is `IsTestSupportProject` and `IsReferenceAssembly`